DiscordWebhooks/DiscordWebhooks.cs

```diff
@@ -59,6 +59,54 @@ public async Task PostMessageAsync(WebhookMessage message)
 
             _ = (HttpWebResponse)(await request.GetResponseAsync());
         }
+
+        public void EditMessage(ulong messageId, WebhookMessage message)
+        {
+            if (message.username != default) { throw new Exception("DiscordWebhooks: username field is not supported for editing messages."); }
+            if (message.avatar_url != default) { throw new Exception("DiscordWebhooks: avatar_url field is not supported for editing messages."); }
+            
+
+            HttpWebRequest request = WebRequest.CreateHttp(WebhookURL + $"/messages/{messageId}");
```

Comment:
The networking code to send messages should also be updated to the latest refactored version